index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="header">
```

Comment:
- [ ] Your suggestions appear like an order, kindly rephrase your suggestion and make it sound polite. You could use kind words like _**'kindy'**_, _**please**_. _**I think...**_ and other kind words.

- This is a polite suggestion: "I think it would be better to use semantic tags" or "Kindly use semantic tags"

- This reads as an order: "Use semantic tags"

#### When making the change to your inline comment, kindly edit the old one. Please don't create a new comment for it. 👍